src/components/SlateEditor/plugins/link/LinkForm.jsx

```diff
@@ -37,23 +39,39 @@ const linkValidationRules = {
   href: { required: true, urlOrNumber: true },
 };
 
-const getLinkFieldStyle = href => {
+const getLinkFieldStyle = (href, node) => {
   const baseHref = href.split(/\?/)[0];
-  if (
-    isNDLAArticleUrl(baseHref) ||
+  const data = node?.data?.toJS() || {};
+  const savedHref = data.href;
+  const isExternalLink = data.resource !== 'content-link';
+
+  if (isExternalLink && href === savedHref) {
+    return css`
+      input {
+        background-color: ${colors.tasksAndActivities.background};
+      }
+    `;
+  } else if (
     isPlainId(baseHref) ||
-    isNDLATaxonomyUrl(baseHref) ||
-    isNDLAEdPathUrl(baseHref) ||
-    isNDLALearningPathUrl(baseHref)
+    (isUrl(baseHref) &&
+      (isNDLAArticleUrl(baseHref) ||
+        isNDLATaxonomyUrl(baseHref) ||
+        isNDLAEdPathUrl(baseHref) ||
+        isNDLALearningPathUrl(baseHref)))
```

Comment:
Er dette egentlig behov for å ha med? Dersom lenka er lagra som en embed er den per def en intern ressurs og desom den er en lagra som en a er den ekstern. Det vi ønsker å oppnå her er å kunne sjå forskjell på lagringa og ikkje om urlen har ndla.no i seg.

Reply:
Det kommer an på om inputfeltet skal endre farge når brukeren redigerer det. Slik det er nå får man indikasjon på om lenken kommer til å bli lagret som en intern ressurs eller ekstern lenke.

Reply:
Eller er det kun viktig hvordan det er lagret? Så ved oppretting av ny lenke vises ingen farge.

Reply:
Eg tenker det kun er viktig korleis det er lagra.